padrino/manifest.yml

```diff
@@ -0,0 +1,3 @@
+---
+applications:
+-  command: bundle exec padrino start -p $PORT
```

Comment:
Did you try it with a name? My thought is that cloud foundry may be ignoring the manifest file since it's invalid. It probably works with the sinatra app because that works with a default command, and it would work the same way without a manifest file.